docs/prow/prow-upgrade.md

```diff
@@ -0,0 +1,51 @@
+# Prow cluster update
+
+In order to get improved/newer versions of prow we need to upgrade prow. The kubernetes teams' prow instance is getting updated via a shell script. This shell script offers a small list of the last pushed container tags. This could reflect only containers of a single day, due to how many times the kubernetes team is actually running their pipeline. This limits us in the versions we can choose from, as we usually don't go from finding out about a new version, to deploying it, within one day. To cherrypick updates, we will need a different process.
+
+In the meantime though we can't not update the cluster and will have to describe a way to do updates. This document aims to provide a description of such an upgrade.
+
+## Update process
+
+To update a Prow cluster follow these steps:
+
+1. Follow [this](./prow-installation-on-forks.md) document for details to set up a Prow cluster.
```

Comment:
I think we should add a step where we need to check the announcements, especially the breaking changes, from here: https://github.com/kubernetes/test-infra/blob/master/prow/ANNOUNCEMENTS.md

Reply:
added this in the first paragraph as a general thing.